app/main.py

```diff
@@ -1,7 +1,43 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+from typing import Dict
+
+import requests
+
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+if not API_KEY:
+    raise ValueError("API_KEY environment variable not set")
+
+
+def get_weather(city: str) -> Dict:
+    """Fetch current weather data for the specified city."""
+    params = {
+        "key": API_KEY,
+        "q": city,
+    }
+    response = requests.get(BASE_URL, params=params)
+    response.raise_for_status()
```

Comment:
Consider adding a try-except block around the request to handle potential network errors or API issues more gracefully. This will allow you to provide a user-friendly error message instead of letting the program crash.